src/seedu/addressbook/commands/Command.java

```diff
@@ -39,12 +39,12 @@ public static String getMessageForPersonListShownSummary(List<? extends ReadOnly
     /**
      * Executes the command and returns the result.
      */
-    public CommandResult execute(){
-        throw new UnsupportedOperationException("This method should be implement in child classes");
-    }
+//    public CommandResult execute(){
```

Comment:
it is better to remove unnecessary code completely rather than retaining commented code.
you can always retrieve it from VCS.

Reply:
Okay noted. Thanks for the comment.